packages/docs/src/scripts/components/ReactPropDoc.jsx

```diff
@@ -69,17 +69,38 @@ class ReactPropDoc extends React.PureComponent {
   }
 
   render() {
+    // Specify ARIA roles attribute for table to make responsive HTML table accessible to screen readers
     return (
-      <tr>
-        <td>
+      <tr role="row">
+        <td role="cell" headers="columnname">
+          <span className="docs_table__column-header ds-u-font-weight--bold" aria-hidden="true">
+            Name
+            <br />
+          </span>
           <code className="ds-u-font-weight--bold">{this.props.name}</code>
           {this.isRequired()}
         </td>
-        <td>
+        <td role="cell" headers="columntype">
```

Comment:
@nichia @davidakennedy Sorry if I'm missing some context here, but whats the point of these hidden `span`s again? Is this unhidden when the table is collapsed?